src/store/doc/index.rst

```diff
@@ -41,10 +41,13 @@ You can find more advanced usage in combination with an Agent using the store fo
 * `Similarity Search with Meilisearch (RAG)`_
 * `Similarity Search with memory storage (RAG)`_
 * `Similarity Search with MongoDB (RAG)`_
+* `Similarity Search with Neo4j (RAG)`_
 * `Similarity Search with Pinecone (RAG)`_
+* `Similarity Search with PSR-6 Cache (RAG)`_
 * `Similarity Search with Qdrant (RAG)`_
 * `Similarity Search with SurrealDB (RAG)`_
-* `Similarity Search with Neo4j (RAG)`_
+
+PS: Both `InMemory` and `PSR-6 cache` vector stores are designed to be primarily used in tests / development environments.
```

Comment:
You can add details. The environment doesn't matter, the data volume is the problem.

```suggestion
PS: Both `InMemory` and `PSR-6 cache` vector stores will load all the data into the memory of the PHP process. They can be used only the amount of data fits in the PHP memory limit, typically for testing.
```